internal/pkg/utils/orderedmap/orderedmap.go

```diff
@@ -50,17 +51,17 @@ func (o *OrderedMap) Clone() *OrderedMap {
 	return deepcopy.Copy(o).(*OrderedMap)
 }
 
-func (o *OrderedMap) DeepCopy(callback deepcopy.TranslateFunc, steps deepcopy.Steps, visited deepcopy.VisitedMap) *OrderedMap {
+func (o *OrderedMap) DeepCopy(callback deepcopy.TranslateFunc, steps deepcopy.Steps, visited deepcopy.VisitedPtrMap) (*OrderedMap, deepcopy.CloneNestedFn) {
```

Comment:
Ako som pisal, pribudol return parameter `CloneNestedFn`.
Najprv musime vratit novu hodnotu `New()`, tu si interne poznacime do `VisitedPtrMap`,
A ked sa vykonava callback `CloneNestedFn`, a narazi sa opat na danu `map`, tak uz ju mame poznacenu a neklonujeme ju znova.

Ak by si sa z tohto popisu stracal :sweat_smile: , tak sa na to mozes pozriet tak, ze test prechadza, alebo to mozem okomentovat na standupe.

Reply:
No hele, myslím že tomuto kupodivu vlastně i rozumím. 😆

Reply:
:+1: